starboard/BUILD.gn

```diff
@@ -39,7 +39,10 @@ group("gn_all") {
   }
 
   if (platform_tests_path == "") {
-    deps += [ ":starboard_platform_tests($starboard_toolchain)" ]
+    deps += [
+      ":starboard_platform_tests($starboard_toolchain)",
+      ":starboard_platform_tests_install($starboard_toolchain)",
```

Comment:
The only target we do this for is `cobalt_install` so that it builds for the default target: https://github.com/youtube/cobalt/blob/37b16ba393d9b2e579881396ed62ade040f2a39e/cobalt/BUILD.gn#L65

I am not sure why the install target for `starboard_platform_tests` is not reachable when building all.

Reply:
we do this also for `player_filter_tests`  which is my reference working  target for `starboard_platform_tests`

Reply:
I see. I think there's something missing in the logic for managing these special starboard targets.

I'm approving for now but will have to think about how we're going to manage these.